datumaro/plugins/accuracy_checker_plugin/details/ac.py

```diff
@@ -3,9 +3,6 @@
 #
 # SPDX-License-Identifier: MIT
 
-from datumaro.util.tf_util import import_tf
```

Comment:
I think this import is supposed to go first, because we need to make sure that we import TF before Accuracy Checker does. Can you suppress the Pylint warning just for this import?

Reply:
It was possible to do this for the specified file.

Reply:
You need to do the same in `representation.py`.

Reply:
Done